src/spec/genre_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../Classes/genre'
+
+describe Genre do
+  context 'Unit tests for Genre Class' do
+    it 'should create a new genre when the class is instantiated' do
+      genre = Genre.new(21, 'Blues')
+      genre.add_item({ one: 1, two: 2 })
+      expect(genre.name).to eq('Blues')
+      expect(genre.items.length).to eq(1)
+    end
```

Comment:
Please add a test to check whether a music genre is added properly. 
hint: First create a genre object then add it to the genre property of the music object and then you can check whether it's an instance of the genre object